app/system/mailer.ts

```diff
@@ -91,4 +114,19 @@ export class Mailer {
         const options = this._buildMailOptionsForExamFeedback(context);
         await this._sendMail(to, options);
     }
+
+    private _buildMailOptionsForSummaryFeedback(
+        context: SummaryFeedbackContext,
+    ): Options {
+        const subject = `Resumen de cursada - Padrón ${context.padron}`;
+        const text = this._render(`emails/notas_examen_plain.html`, context);
+        const html = this._render(`emails/notas_examen.html`, context);
+        const replyTo = this._replyTo;
+        return { subject, text, html, replyTo };
+    }
+
+    async sendSummaryFeedback(context: SummaryFeedbackContext, to: string) {
+        const options = this._buildMailOptionsForSummaryFeedback(context);
+        await this._sendMail(to, options);
```

Comment:
El MailerClient no está configurado porque al final se mandan por el script de Google. Haria que devuelva las opciones y se siga desde el script.

```suggestion
        return options;
```